odoo/modules/migration.py

```diff
@@ -75,6 +81,7 @@ def get_scripts(path):
             self.migrations[pkg.name] = {
                 'module': get_scripts(get_resource_path(pkg.name, 'migrations')),
                 'maintenance': get_scripts(get_resource_path('base', 'maintenance', 'migrations', pkg.name)),
+                'migrations': get_scripts(_get_migrations_path(pkg.name)),
```

Comment:
for master only: support 'upgrades' along with 'migrations' for the `module` and `maintenance` mappings, for consistency